### ISSUE TITLE
Can you add the "SWAP U and V" basic transformation to the collection of CUDA accelerated effects?

### ISSUE THREAD

**Issue by Cuberlain** [rigaya/NVEnc#266]
Hello Rigaya!

Can you implement the "SWAP U and V" basic transformation to the collection of CUDA accelerated effects?

If there are rotate and transpose CUDA accelerated effects, than this basic transformation would be important too in the empire of YUV colors...

Thank you!

**Comment by rigaya** [comment]
I'll add the feature to [--vpp-tweak](https://github.com/rigaya/NVEnc/blob/master/NVEncC_Options.en.md#--vpp-tweak-param1value1param2value2) in the next release, adding new option "swapuv".

**Comment by Cuberlain** [comment]
Thank you very much Rigaya!

**Comment by rigaya** [comment]
The uvswap feature has been added to [--vpp-tweak](https://github.com/rigaya/NVEnc/blob/master/NVEncC_Options.en.md#--vpp-tweak-param1value1param2value2) in [NVEnc 5.16](https://github.com/rigaya/NVEnc/releases/tag/5.16).

**Comment by Cuberlain** [comment]
Hello Rigaya!

I tried out the swap UV option.

I used this :  --vpp-tweak swapuv=true
But the colors remained the same on my YV12 videos. :-(

**Comment by Selur** [comment]
same here, doesn't seem like swapuv is doing anything aside from changing the encoder output message
without: tweak: brightness 0.00, contrast 1.00, saturation 1.00, gamma 1.00, hue 0.00
with: tweak: brightness 0.00, contrast 1.00, saturation 1.00, gamma 1.00, hue 0.00, swapuv
call I used was:
```
NVEncC64 --avhw  -i "F:\TestClips&Co\files\file_1.mp4" --fps 25.000 --codec h265 --profile main10 --level auto --tier high --sar 1:1 --lookahead 16 --output-depth 10 --vbrhq 0 --vbr-quality 23.00 --max-bitrate 240000 --aq --aq-strength 5 --gop-len 240 --ref 7 --multiref-l0 7 --multiref-l1 7 --nonrefp --bframes 0 --no-b-adapt --mv-precision Q-pel --preset default --colormatrix bt470bg --vpp-resize default --output-res 640x352 --vpp-gauss disabled --vpp-tweak brightness=0,contrast=1,gamma=1,saturation=1,hue=0,swapuv=true/false --cuda-schedule sync --output "E:\Temp\file_1_2020-11-09@16_16_07_1210_01.265
```

**Comment by rigaya** [comment]
Thanks for letting me know about this, I've fixed this in [NVEnc 5.20](https://github.com/rigaya/NVEnc/releases/tag/5.20).
